src/meta-srv/src/metasrv.rs

```diff
@@ -102,20 +106,51 @@ pub struct MetaSrv {
     election: Option<ElectionRef>,
     meta_peer_client: Option<MetaPeerClient>,
     lock: Option<DistLockRef>,
+    procedure_manager: ProcedureManagerRef,
```

Comment:
Could procedure manager be an optional field? So we don't need to create it if we don't enable the experimental procedure framework.

Reply:
I plan to always enable procedure in distributed mode, because it can't be worse than the current implementation, which is rather fragile.